blocks/init/src/Blocks/components/button/manifest.json

```diff
@@ -20,6 +20,10 @@
 		"buttonId": {
 			"type": "string"
 		},
+		"buttonType": {
+			"type": "string",
+			"default": ""
```

Comment:
is there a need for this `""` ?

Reply:
Not really, could be omitted... pushed a commit